static/include/components/apis/overrides/protos/button.DoCommand.md

```diff
@@ -1,3 +1,4 @@
 Execute model-specific commands that are not otherwise defined by the component API.
-For built-in models, model-specific commands are covered with each model's documentation.
-If you are implementing your own button and add features that have no built-in API method, you can access them with `DoCommand`.
+Most models do not implement `DoCommand`.
+Any available model-specific commands should be covered in the model's documentation.
+If you are implementing your own button and want to add features that have no corresponding built-in API method, you can implement them withu can implement them withu can implement them with [`DoCommand`](/dev/reference/sdks/docommand/).
```

Comment:
```suggestion
If you are implementing your own button and want to add features that have no corresponding built-in API method, you can implement them with [`DoCommand`](/dev/reference/sdks/docommand/).
```

Reply:
Oh man. Do you ever have this same sort of search-replace bug, where it applies something in addition/duplicating part of what you put in?